src/routes/plugin.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response, Router } from "express";
+import { PluginState, setPlugin } from "sqlite/plugins";
+
+const router = Router();
+
+router.get('/', async (req, res) => {
+  res.json({
+    status: "success",
+  });
+});
+
+router.post('/setPlugin', async (req: Request, res: Response) => {
+  const plugin  =  req.body.plugin;
+  const state = req.body.state ?? PluginState.DISABLED;
```

Comment:
Since we have only two states (enable / disable), I'd prefer organising it as two routes:
/enable/:name
/disable/name
so there's no potential mistake of providing the wrong state.
But your approach though can provide us some versatility in perspective if we want those states to be more complex.

all the rest looks good to me, @HariHive , thank you!